linode/manager
# upcoming: [DI-18311] - Add CloudPulse feature flag and landing page to Cloud Manager

## Description 📝
Added cloudpulse section to cloud manager
Added feature flag control for cloudpulse section
Added an empty paper component in the cloudpulse section

## Changes  🔄
List any change relevant to the reviewer.
-Added cloudpulse section cloud manager
-Added feature flag control for cloudpulse section
-Added empty paper component in the cloudpulse section

## Preview 📷
**Include a screenshot or screen recording of the change**

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-04-22 at 18 04 10](https://github.com/linode/manager/assets/159890961/5e3bb1df-f768-42f0-a1a8-fe04ae24a77b)| ![image](https://github.com/linode/manager/assets/159890961/c9a2683c-f9bd-4233-85e0-01cd840dc8ce)
|

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Follow cloud manager standard procedures

### Reproduction steps
(How to reproduce the issue, if applicable)
- N/A

### Verification steps
(How to verify changes)
- Navigate to cloud manager and once feature flag of CloudPulse is enabled then navigate to the Monitor section in the sidebar for the CloudPulse section.
## As an Author I have considered 🤔

*Check all that apply*
